client/src/app/site/interaction/components/applause-bar-display/applause-bar-display.component.html

```diff
@@ -1,9 +1,3 @@
 <div class="bar-wrapper">
-    <os-progress class="progress-bar" [value]="percent">
-        <div class="level-indicator">
-            <div class="level">
-                <b *ngIf="showLevel && hasLevel" [@fade]="'in'">{{ level }}</b>
-            </div>
-        </div>
```

Comment:
@emanuelschuetze why would you remove that?

Reply:
we need one level indicator only -> the badge